File: solved.ac/source/1110.js
let fs = require("fs");
let input = fs.readFileSync("./dev/stdin").toString().split(" ");

let num = Number(input);

let cycle = 0;
let temp = num;
let step = 0;
let isNotSame = true;

while (isNotSame) {
  cycle++;

  if (temp < 10) {
    step = temp % 10;
  } else {
    step = Math.floor(temp / 10) + (temp % 10);
  }
  /*
  step = temp % 10;
  if (temp >= 10) {
    step += Math.floor(temp / 10);
  }
  */

  temp = (temp % 10).toString() + (step % 10).toString();
  temp = Number(temp);

  if (temp === num) {
    isNotSame = false;
  }
}

console.log(cycle);

File: solved.ac/source/1110.py
n = num = int(input())
count = 0
while True:
    ten = n // 10
    one = n % 10
    total = ten + one
    count += 1
    n = int(str(n % 10) + str(total % 10))
    if(num == n):
        break
print(count)